prof.py

```diff
@@ -90,33 +90,34 @@ def get_bin_vals(self, counts_img, bkg_img,
                 j, k = pixel[0], pixel[1]
                 if exp_img_data[i][j, k] != 0:
                     exp_val = exp_img_data[i][j, k]
+                    exp_val_bkg = exp_img_data[i][j, k] * \
+                                  bkg_img_hdr[i]['EXPOSURE'] / \
+                                  counts_img_hdr[i]['EXPOSURE']
                     npix += 1
                 else:
                     continue
                 raw_cts += counts_img_data[i][j, k]
                 net_cts += counts_img_data[i][j, k] - \
                     bkg_img_data[i][j, k] * bkg_corr_i
-                bkg_cts += bkg_img_data[i][j, k]
-                raw_rate += counts_img_data[i][j, k] / exp_val
-                net_rate += (counts_img_data[i][j, k] -
-                    bkg_img_data[i][j, k] * bkg_corr_i) / exp_val
-                bkg_rate += bkg_img_data[i][j, k] * \
-                    bkg_corr_i / exp_val
-                err_raw_rate_sq += counts_img_data[i][j, k] / exp_val**2
-                err_net_rate_sq += (counts_img_data[i][j, k] + \
-                    bkg_img_data[i][j, k] * bkg_corr_i**2) / exp_val**2
-                err_bkg_rate_sq += bkg_img_data[i][j, k] * \
-                    bkg_corr_i**2 / exp_val**2
+                bkg_cts += bkg_img_data[i][j, k] * bkg_corr_i
+
+                err_bkg_cts_sq += bkg_img_data[i][j, k] * bkg_corr_i**2
+
+                exp_raw += exp_val
+                exp_bkg += exp_val_bkg
```

Comment:
Can be changed to `exp_bkg = exp_val / bkg_corr_i`, and we can get rid of exp_val_bkg above.